parse/ValueRefPythonParser.cpp

```diff
@@ -741,7 +741,7 @@ void RegisterGlobalsValueRefs(boost::python::dict& globals, const PythonParser&
             {"Floor", ValueRef::OpType::ROUND_DOWN},
             {"Sign",  ValueRef::OpType::SIGN}})
     {
-        std::function<boost::python::object(const boost::python::tuple&, const boost::python::dict&)> f_insert_abs = [&parser, op](const boost::python::tuple& args, const boost::python::dict& kw) { return insert_1arg_(parser, op.second, args, kw); };
+        const auto f_insert_abs = [&parser, op](const boost::python::tuple& args, const boost::python::dict& kw) { return insert_1arg_(parser, op.second, args, kw); };
```

Comment:
why const?

Reply:
Why not as it doesn't intended to change?

Reply:
it's a local that you're making a copy of when passing. might as well pass with `std::move` or directly pass the temporary. making it const doesn't really do anything, but would prevent the move or copy elision presumably, if they are possible.

Reply:
I removed separate variable entirely